.github/workflows/_rocm-test.yml

```
@@ -61,6 +65,17 @@ jobs:
       - name: Setup ROCm
         uses: ./.github/actions/setup-rocm
 
+      - name: configure aws credentials
+        id: aws_creds
+        uses: aws-actions/configure-aws-credentials@v1.7.0
```

1.7.0 were release 2+ years ago, is there a reason not to migrate to the latest release?
```suggestion
        uses: aws-actions/configure-aws-credentials@v4
```